doc/WebOnChain.plantuml

```diff
@@ -0,0 +1,186 @@
+@startuml
+participant "End user" as user
+participant "Browser" as ui
+participant "Thyra server" as thyra
+participant "Hosting websites" as web
+participant "Wallet" as w
+participant "DNS" as dns
+participant "Event Manager" as em
+participant "Storage" as storage
+participant "Node (API)" as node
+
+user -> ui: click on install 'Web on-chain' plug-in
+ui -> thyra: GET /plugin/webOnChain
+thyra -> web: GET /plugin/webOnChain
```

Comment:
So you need to have the webOnChain plug-in to be installed to install the webOnChain plug-in ?

Reply:
Hmm, no. I wanted to show that somehow when the user click on install (so from ui -> thyra) the name, port or sth else must be clearly identified and known as a reference to the web on-chain plug-in so that the installation starts.

Reply:
I think I mistaken the installation process....to install the plug-in we need to download the packages (installation script, no sure about the best vocabulary here). So the steps here aren't relevant. 
Will update the diagrams with a suggestion. Thanks :)